stdlib/crypto/src/main/ballerina/src/crypto/crypto.bal

```diff
@@ -445,12 +473,13 @@ function externDecryptRsaEcb(byte[] input, PrivateKey|PublicKey key, handle padd
 } external;
 
 # Returns AES CBC decrypted value for the given AES CBC encrypted data.
+# ```ballerina byte[]|crypto:Error plainText = crypto:decryptAesCbc(cipherText, key, initVector); ```
 #
 # + input - The content to be decrypted
 # + key - Encryption key
 # + iv - Initialization vector
 # + padding - The padding
-# + return - Decrypted data or `Error` if key is invalid
+# + return - Decrypted data or `crypto:Error` if key is invalid
```

Comment:
```suggestion
# + return - Decrypted data or `crypto:Error` if key is invalid
```
Please refer the template for the correct format.

Reply:
Corrected.